src/main/java/prefs/Preferences.java

```diff
@@ -135,35 +253,57 @@ public void setLastOpenBoard(String board) {
             
             setLastOpenBoard(boardNames.get(index));
         }
-        
+        save();
```

Comment:
Should this be inside the if clause above? I don't think we need to save if we don't change anything.